container_api/src/container.rs

```diff
@@ -283,4 +367,23 @@ pub mod tests {
             )
         );
     }
+
+    #[test]
+    fn test_rpc_info_instances() {
+        let config = load_configuration::<Configuration>(test_toml()).unwrap();
+
+        // TODO: redundant
+        let mut container = Container::with_config(config.clone());
+        container.dna_loader = test_dna_loader();
+        assert!(container.load_config(&config).is_ok());
+
+        let instance_config = &config.interfaces[0];
+        let dispatcher = container.make_dispatcher(&instance_config);
+        let io = dispatcher.io;
+
+        let request = r#"{"jsonrpc": "2.0", "method": "info/instances", "params": null, "id": 1}"#;
```

Comment:
JsonString?

Reply:
Not in this case, it's a test, the string is being created raw, and it's being used in a function that only takes String